bin/less2sass

```diff
@@ -32,26 +39,46 @@ if ARGV.size < 1
 else
   input_file = ARGV[0]
 
-  src = File.read input_file
-  sass = Less2Sass.convert src
+  if input_file[-5, 5] == ".less"
+    src = File.read input_file
+    sass = Less2Sass.convert src
 
-  if options[:outputfile]
-    output_file = options[:outputfilename]
-  else
-    output_file = input_file.gsub("less", "scss")
-  end
+    if options[:print]
+      puts sass
+      exit
+    end
 
-  File.open output_file, 'w' do |file|
-    file << sass
-  end
+    if options[:outputfile] && options[:outputfilename][-5, 5] == ".scss"
+      output_file = options[:outputfilename]
+    else
+      output_file = input_file.gsub(".less", ".scss")
+    end
+
+    File.open output_file, 'w' do |file|
+      file << sass
+    end
 
-  puts "Done: #{output_file}"
+    if File.file?(output_file)
+      if options[:delete]
+        if @os.include?('mswin') || @os.include?('win') || @os.include?('mingw')
```

Comment:
Good catch realizing that `rm` wouldn't work on Windows, but is there any reason you didn't use `File.delete`?